docs/deployment/getting_started_locally_with_extensions.md

```diff
@@ -0,0 +1,160 @@
+# Deploying Gardener locally and enable provider-extensions
+
+This document will walk you through deploying Gardener on your local machine and bootstrap your own seed clusters on a existing Kubernetes cluster.
+It is supposed to run your local Gardener developments on a real infrastructure. For running Gardener only entirely local, please check the [getting started locally](getting_started_locally.md) docs.
+If you encounter difficulties, please open an issue so that we can make this process easier.
+
+Gardener runs in any Kubernetes cluster.
+In this guide, we will start a [KinD](https://kind.sigs.k8s.io/) cluster which is used as garden cluster. Any Kubernetes cluster could be used as seed clusters in order to support provider extensions (please refer to the [architecture overview](../concepts/architecture.md)). This guide is tested for using Kubernetes Clusters provided by Gardener, AWS, Azure and GCP as seed so far.
+
+Based on [Skaffold](https://skaffold.dev/), the container images for all required components will be built and deployed into the clusters (via their [Helm charts](https://helm.sh/)).
+
+![Architecture Diagram](content/getting_started_locally_with_extensions.png)
```

Comment:
Comment to update the architecture diagram as soon as everyone agreed with the chosen approach.

Reply:
Nice!